medical-portal/src/UI/src/assets/chefs/initChefsForm.js

```diff
@@ -280,11 +280,45 @@ function getChefsBundle() {
 }
 
 function putChefsSubmission(status = "Draft") {
+  const flattenedComponents = utils.flattenComponents(
+    Formio.forms[window.currentFormId].components,
+  );
```

Comment:
this is duplicate code, you could make this a method e.g. "this.getFlattenedComponents()"